domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTransactionsIT.java

```diff
@@ -521,11 +516,6 @@ public PreparedQuery<RowSet<Row>> preparedQuery(String sql, PrepareOptions optio
       return conn.preparedQuery(sql, options);
     }
 
-    @Override
-    public void close(Handler<AsyncResult<Void>> handler) {
-      close().onComplete(handler);
-    }
-
```

Comment:
Really annoying that Vert.x changes interfaces in what appears as bug fix release. Not following semantic versioning policy at all.